### PR TITLE
[ORPH-32] Add capability to pass any additional docker options to the…

### PR DESCRIPTION
**Pull request by carter-cundiff**
… docker build

### CODE REVIEW

**Review by peter-mcclonski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -229,6 +229,32 @@ Specifies target architectures when using the multiplatform build CLI goal.
...
+
+```xml
+<configuration>
+    <buildOptions>
+        <buildOption>--no-cache</buildOption>
+        <buildOption>--label</buildOption>
+        <buildOption>test-label=test-value</buildOption>
+        <buildOption>--progress</buildOption>
+        <buildOption>auto</buildOption>
+        <buildOption>--output=type=docker</buildOption>
```
Comment (peter-mcclonski) [inline README.md:250]: A: output=type=docker? Should this be output-type=docker?

Comment (carter-cundiff) [reply README.md:250]: I stole it from the docker help command. Was really just looking for args that wouldn't change the current functionality:
```
--load                          Shorthand for "--output=type=docker"
```

Comment (carter-cundiff) [reply README.md:250]: Better link: https://docs.docker.com/engine/reference/commandline/buildx_build/#load

Comment (peter-mcclonski) [reply README.md:250]: LGTM

**Pull request merged**